site-src/api-types/grpcroute.md

```diff
@@ -0,0 +1,216 @@
+!!! info "Experimental Channel"
+
+    The `GRPCRoute` resource described below is currently only included in the
+    "Experimental" channel of Gateway API. For more information on release
+    channels, refer to the [related documentation](https://gateway-api.sigs.k8s.io/concepts/versioning).
+
+# GRPCRoute
+
+[GRPCRoute][grpcroute] is a Gateway API type for specifying routing behavior
+of gRPC requests from a Gateway listener to an API object, i.e. Service.
+
+## Spec
+
+The specification of a GRPCRoute consists of:
+
+- [ParentRefs][parentRef]- Define which Gateways this Route wants to be attached
+  to.
+- [Hostnames][hostname] (optional)- Define a list of hostnames to use for
+  matching the Host header of gRPC requests.
+- [Rules][grpcrouterule]- Define a list of rules to perform actions against
+  matching gRPC requests. Each rule consists of [matches][matches],
+  [filters][filters] (optional), and [backendRefs][backendRef] (optional)
+  fields.
+
+The following illustrates a GRPCRoute that sends all traffic to one Service:
+![grpcroute-basic-example](/images/grpcroute-basic-example.png)
```

Comment:
Can you add a markdown comment to a link to a source where this graphic can be edited/modified in the future if needed?

Reply:
Done.